app/modules/sightings/models.py

```diff
@@ -30,6 +30,8 @@ class Sighting(db.Model, FeatherModel):
 
     assets = db.relationship('SightingAssets')
 
+    _featured_asset_guid = db.Column(db.GUID, default=None, nullable=True)
```

Comment:
should this have `db.ForeignKey('asset.guid')` as well?
i wonder if there is even a fancier way to constrain it to be within `assets` as well (assuming that is part of the design!  unsure?)... that part is definitely reaching my limits of the orm magic.

Reply:
I would venture that it doesn't need it. There is in-method validation of the _featured_asset_guid before you can set or get it and it's prepended with _ to make it private. No need to have another link across to the asset table.